Pod/Classes/PINRemoteImageMacros.h

```diff
@@ -35,10 +35,14 @@
 #define PINImage     UIImage
 #define PINImageView UIImageView
 #define PINButton    UIButton
+#define PINNSOperationSupportsQOS (kCFCoreFoundationVersionNumber >= kCFCoreFoundationVersionNumber_iOS_8_0)
+#define PINNSURLSessionTaskSupportsPriority (kCFCoreFoundationVersionNumber >= kCFCoreFoundationVersionNumber_iOS_8_0)
```

Comment:
Wait, this is compile time, no? That means if will always evaluate to true as long as it's compiled with a version of the SDK that's greater than or equal to 8. However, when it's run on iOS 7, it will still be true and cause crashes?